public/stylesheets/question.css

```diff
@@ -0,0 +1,113 @@
+.prompt {
+    /* font-family: Poppins; */
+    font-style: normal;
+    font-weight: bold;
+    font-size: 30px;
+    color: #1652A3;
+}
+
+
+.jumbotron {
```

Comment:
If these jumbotron styles are going to be re-usable across many different pages, maybe we should move them into something like a "common.css" file that everyone can look to for shared styles across pages.

And then just keep question-specific styling here.